apps/prairielearn/src/pages/instructorJobSequence/instructorJobSequence.js

```diff
@@ -1,17 +1,18 @@
 // @ts-check
-const ERR = require('async-stacktrace');
-import * as express from 'express';
-import * as error from '@prairielearn/error';
+const asyncHandler = require('express-async-handler');
+import { Router } from 'express';
+import { HttpStatusError } from '@prairielearn/error';
 
-import * as serverJobs from '../../lib/server-jobs-legacy';
+import { getJobSequenceWithFormattedOutput } from '../../lib/server-jobs-legacy';
 
-const router = express.Router();
+const router = Router();
 
-router.get('/:job_sequence_id', function (req, res, next) {
-  const job_sequence_id = req.params.job_sequence_id;
-  const course_id = res.locals.course ? res.locals.course.id : null;
-  serverJobs.getJobSequenceWithFormattedOutput(job_sequence_id, course_id, (err, job_sequence) => {
-    if (ERR(err, next)) return;
+router.get(
+  '/:job_sequence_id',
+  asyncHandler(async (req, res) => {
+    const job_sequence_id = req.params.job_sequence_id;
+    const course_id = res.locals.course ? res.locals.course.id : null;
```

Comment:
As long as we're changing things here:

```suggestion
    const course_id = res.locals.course?.id ?? null;
```